test_integration/m1_facilitator/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+
+{
+    "compilerOptions": {
+        "target": "es6",
+        "module": "commonjs",
+        "alwaysStrict": true,
+        "sourceMap": true,
+        "resolveJsonModule": true,
+        "outDir": "./test_dist",
+        "declaration": true,
+        "skipLibCheck": true,
+        "esModuleInterop": true
+    },
+    "exclude": [
+        "node_modules"
+    ]
```

Comment:
Why it's needed? Can we use the root tsconfig.json.

Reply:
Removed ts config. Initially, it was needed for `tsc`